### PR TITLE
Video encodings html cleaner

### PR DESCRIPTION
**Pull request by NathanielBennett**
This deals with the problem of having newlines appearing in octopus video encoding urlls. See example of api content with the resultant broken dom below. 

Api Content: 
broken-video-api

Broken Dom: 
broken-dom

Fixed Dom:
fixed-dom

### CODE REVIEW

File: applications/test/MediaControllerTest.scala [hunk 1]
```diff
@@ -6,6 +6,7 @@ import org.scalatest.{DoNotDiscover, Matchers, FlatSpec}
 @DoNotDiscover class MediaControllerTest extends FlatSpec with Matchers with ConfiguredTestSuite {
 
   val videoUrl = "uk/video/2012/jun/26/queen-enniskillen-northern-ireland-video"
+  val videoUrlWithDodgyOctpusUrl = "football/video/2015/feb/10/manchester-united-louis-van-gaal-long-ball-video"
```
Comment (Jholder112233) [inline applications/test/MediaControllerTest.scala]: Probably worth fixing up 'Octpus'

Comment (NathanielBennett) [inline applications/test/MediaControllerTest.scala]: Yeah - that's in hand too.

**Comment by Jholder112233** [comment]
Otherwise this looks ok to me :+1:

**Comment by gklopper** [comment]
:+1:

**Pull request merged**